docs/layouts/index.html

```diff
@@ -92,6 +90,16 @@ <h2 class="mt-6 text-xl mx-auto text-center font-bold text-gray-400 leading-6 tr
     </section>
   </div>
 </div>
+<div>
+  <div class="mt-6 grid grid-cols-1 gap-4 w-full">
+    <h2 class="mt-6 text-xl mx-auto text-center font-bold text-gray-400 leading-6 tracking-tighter bg-clip-text text-transparent bg-gradient-to-b from-gray-900 to-gray-600 dark:from-gray-100 dark:to-gray-400" style="max-width: 800px">
+     We are a <a href="https://www.cncf.io/">Cloud Native Computing Foundation</a> Sandbox Project
+     <br>
```

Comment:
I have no idea how to do this without a bunch of `<br>`s to make spacing nice.

Reply:
I might have also done the same. If the footer looks good to you, It's good then. Also, should we also give credit to Hextra by adding the `Powered by Hextra` text in footer?

Reply:
Yeah, we should give credit to Hextra, you're right. I'll turn back on the "powered by" in the config in a second :)